components/public-api/gitpod/v1/workspace.proto

```diff
@@ -278,3 +615,59 @@ message WorkspaceEnvironmentVariable {
   string name = 1;
   optional string value = 2;
 }
+
+message WorkspaceClass {
+  // id is the unique identifier of the workspace class
+  string id = 1;
+
+  // display_name is the human readable name of the workspace class
+  string display_name = 2;
+
+  // description is a human readable description of the workspace class
+  string description = 3;
+
+  // is_default indicates if this workspace class is the default one
+  bool is_default = 4;
+}
+
+message WorkspaceSpec {
```

Comment:
we should not have it, instead create should list fileds should be provided on create, and update should list fields which can be updated on workspace, see https://www.notion.so/gitpod/Stable-Public-API-Project-Spec-f5225e862cc94b22a659094d1e7a4570?pvs=4#c17532fbbf884e0798c66abb9d6d93b6